src/pages/DistributionsPage/DistributionForm.tsx

```diff
@@ -292,259 +300,374 @@ export function DistributionForm({
   };
 
   return (
-    <Box css={twoColStyle}>
+    <TwoColumnLayout>
       <form onSubmit={handleSubmit(onSubmit)}>
-        <Box css={headerStyle}>Gift Circle</Box>
-        <Box css={{ display: 'flex', justifyContent: 'center', pt: '$lg' }}>
-          <Box css={{ mb: '$lg', mt: '$xs', mr: '$md', width: '100%' }}>
-            <FormControl fullWidth>
-              <Box css={vaultInputStyles}>Select Vault</Box>
+        <Panel css={{ padding: '16px', minHeight: '147px' }}>
+          <Text h2 css={headerStyle}>
+            Gift Circle
+          </Text>
+          <TwoColumnLayout css={{ pt: '$md' }}>
+            <Box css={{ width: '100%' }}>
+              <FormControl fullWidth>
+                <Controller
```

Comment:
just FYI, react-hook-form also has a `useController` hook that you can use instead of this component. I think it's nicer because it produces less nesting. no need to change it in this PR though.

Reply:
created an issue for this
https://github.com/coordinape/coordinape/issues/1168